jellyfin_kodi/jellyfin/http.py

```diff
@@ -49,48 +49,60 @@ def stop_session(self):
         except Exception as error:
             LOG.warning("The requests session could not be terminated: %s", error)
 
-    def _replace_user_info(self, string):
+    def _replace_user_info(self, config, string):
 
         if '{server}' in string:
-            if self.config.data.get('auth.server', None):
-                string = string.replace("{server}", self.config.data['auth.server'])
+            if config.get('auth.server', None):
+                string = string.replace("{server}", config['auth.server'])
             else:
                 LOG.debug("Server address not set")
 
         if '{UserId}'in string:
-            if self.config.data.get('auth.user_id', None):
-                string = string.replace("{UserId}", self.config.data['auth.user_id'])
+            if config.get('auth.user_id', None):
+                string = string.replace("{UserId}", config['auth.user_id'])
             else:
                 LOG.debug("UserId is not set.")
 
         return string
 
-    def request(self, data, session=None):
-
-        ''' Give a chance to retry the connection. Jellyfin sometimes can be slow to answer back
-            data dictionary can contain:
-            type: GET, POST, etc.
-            url: (optional)
-            handler: not considered when url is provided (optional)
-            params: request parameters (optional)
-            json: request body (optional)
-            headers: (optional),
-            verify: ssl certificate, True (verify using device built-in library) or False
-        '''
-        if not data:
-            raise AttributeError("Request cannot be empty")
-
-        data = self._request(data)
-        LOG.debug("--->[ http ] %s", JsonDebugPrinter(data))
-        retry = data.pop('retry', 5)
+
+    def get_handler_url(self, handler):
+        server = self.config.data.get("auth.server", "")
```

Comment:
Looks like the rest of the instances of this have had the self and .data removed. Should this one not also have that?

Reply:
`_replace_user_info` is getting `config` as a parameter, `get_handler_url` is not.